graphql/types/Profile.graphql

```diff
@@ -1,3 +1,5 @@
+scalar Upload
```

Comment:
could be added to `schema.graphql` to be used in all sub types

Reply:
It's global